backend/src/controllers/user.go

```diff
@@ -33,3 +35,29 @@ func (u *UserController) GetAllUsers(c *fiber.Ctx) error {
 
 	return c.Status(fiber.StatusOK).JSON(users)
 }
+
+// GetUser godoc
+//
+// @Summary		Gets specific user
+// @Description	Returns specific user
+// @ID			get-user
+// @Tags      	user
+// @Produce		json
+// @Success		200	  {object}	  models.User
```

Comment:
nit: address the other status codes and corresponding messages here

Reply:
done